class-wc-gateway-komoju.php

```diff
@@ -119,9 +119,11 @@ public function process_payment( $order_id ) {
 		include_once( 'includes/class-wc-gateway-komoju-request.php' );
 		$order          = wc_get_order( $order_id );
 		$komoju_request = new WC_Gateway_Komoju_Request( $this );
+		$payment_method = sanitize_text_field($_POST['komoju-method']);
```

Comment:
Using the [`sanitize_text_field`](https://developer.wordpress.org/reference/functions/sanitize_text_field/) function to strip out potentially bad data. Technically the `komoju-method` field is the value from the select dropdown and not "true" user input, but it seemed the closest out of the [possible options](https://developer.wordpress.org/plugins/security/securing-input/#sanitizing-the-data)